_dashboards/visualize/maps-stats-api.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: default
+title: Maps Stats API
+nav_order: 20
+grand_parent: Building data visualizations
+parent: Using coordinate and region maps 
+has_children: false
+---
+
+# Maps Stats API
+Introduced 2.7
+{: .label .label-purple }
+
+When you create and save a [map]({{site.url}}{{site.baseurl}}/dashboards/visualize/maps/) in OpenSearch Dashboards, the map becomes a saved object of type `map`. The Maps Stats API provides information about such saved objects in OpenSearch Dashboards. 
+
+#### Example request
+
+You can access the Maps Stats API by providing its URL address in the following format:
+
+```
+<opensearch-dashboards-endpoint-address>/api/maps-dashboards/stats
+```
+
+The OpenSearch Dashboards endpoint address may contain a port number if it is specified in the OpenSearch configuration file. The specific URL format depends on the type of OpenSearch deployment and the network environment in which it is hosted.
+{: .note}  
+
+You can query the endpoint in two ways:
+  
+  - By accessing the endpoint address (for example`http://localhost:5601/api/maps-dashboards/stats`) in a browser
+
+  - By using the `curl` command in the terminal:
```

Comment:
Should there be a line space between lines 31 and 32?

Reply:
No, this renders with correct spacing automatically.